packages/cf-util-http/test/http.js

```diff
@@ -1,94 +1,85 @@
-import { createFakeServer } from '../../cf-test-server/src/index';
-import http from '../../cf-util-http/src/index';
+import * as http from '../../cf-util-http/src/index';
 
-describe.skip('request', () => {
-  test('should call the success handler on success', done => {
-    const server = createFakeServer();
+afterEach(() => fetch.mockClear());
 
-    server.respondWith(
-      'DELETE',
-      '/posts/3',
-      200,
-      { 'Content-Type': 'application/json' },
-      {
+describe('request', () => {
+  test('should call the success handler on success', done => {
+    fetch.mockResponse(
+      JSON.stringify({
         message: 'Hello World'
+      }),
+      {
+        headers: { 'Content-Type': 'application/json' },
+        status: 200
       }
     );
 
-    http.request('DELETE', '/posts/3', null, (err, res) => {
+    http.request('DELETE', '/posts/3', undefined, (err, res) => {
```

Comment:
`null` is defined, `undefined` is not. Default argument values won't work if you passed in null. This is a slight API change but that's ok, no one uses `request()` directly.